src/cts/src/HTreeBuilder.cpp

```diff
@@ -346,7 +335,7 @@ void HTreeBuilder::computeLevelTopology(unsigned level,
   logger_->report("    Sinks per sub-region: {}", numSinksPerSubRegion);
   logger_->report("    Sub-region size: {:.4f} X {:.4f}", width, height);
 
-  const unsigned minLength = minLengthSinkRegion_ / 2;
```

Comment:
@maliberty if fake lutentries is triggered it sets minLengthSinkRegion_ = 1 on line 265 when you divide by 2 it's integer divided into 0

```
        unsigned minIndex = 1;
        techChar_->createFakeEntries(minLengthSinkRegion_, minIndex);
        minLengthSinkRegion_ = 1;
```

Reply:
I don't see any divide by two on the lines you mention.

Reply:
It's in the code block above my first comment?

const unsigned minLength = minLengthSinkRegion_ / 2;